MIP/mip-21/README.md

```diff
@@ -0,0 +1,70 @@
+# MIP-21: Bridge Based on Attestors
+- **Description**: Proposal for simplifying the bridge mechanism using an attestor-based approach inspired by USDC's CCTP, reducing friction, cost, and complexity for Layer 2 onboarding.
+- **Authors**: [Primata](mailto:primata@movementlabs.xyz)
+
+## Abstract
+
+A bridge mechanism is essential for any Layer 2 solution, serving as the user's first interaction with the network. This process must be smooth and frictionless, ensuring that dealing with tokens on L1 and bridging to native gas tokens on L2 does not overcomplicate the user experience. The bridge must balance user onboarding, cost-efficiency, and simplicity, while minimizing transaction complexity. We propose a bridge based on attestors, inspired by the Cross-Chain Transfer Protocol (CCTP) by USDC, which offers an elegant solution to these challenges.
+
+## Motivation
+
+Our current bridge solution has been increasing in complexity with each iteration. Originally requiring three on-chain transactions, it now requires five. Additionally, the bridge does not currently support gas charging mechanisms to prevent exploits, and ongoing discussions are leaning toward adding further components, making it even more complex and computationally expensive.
```

Comment:
Reference where the solution with 3 and 5 steps is missing

Reply:
woops its actually 4 onchain transactions:
1. initiate bridge
2. lock bridge
3. complete bridge source chain
4. complete bridge target chain

Reply:
ok fixed.